### PR TITLE
Adding intial code to add openshift template capability

### LINKED ISSUE #9: Need a way to process OpenShift Templates

### PR DESCRIPTION
**Pull request by etsauer**
resolves #9

### CODE REVIEW

File: examples/oc-template/dash.yaml
Comment (darthlukan) [inline examples/oc-template/dash.yaml]: Typo, `verison` != `version`

**Comment by etsauer** [comment]
@darthlukan thanks, fixed.

**Comment by tylerauerbeck** [comment]
LGTM when running `go run cmd/dash.go -i examples/oc-template/`. 

Before we merge, could we fix the format of the example inventory underneath `examples/default`? It currently errors out because I don't believe it follows the appropriate format.

**Review by tylerauerbeck — APPROVED** [review]
LGTM when running go run cmd/dash.go -i examples/oc-template/.

Before we merge, could we fix the format of the example inventory underneath examples/default? It currently errors out because I don't believe it follows the appropriate format.

**Comment by tylerauerbeck** [comment]
After merging #3 we'll need a rebase, but should be good after that.

**Review by tylerauerbeck — APPROVED** [review]
LGTM

**Pull request merged**
